src/views/SpecialResolution.vue

```diff
@@ -350,7 +350,7 @@ export default class SpecialResolution extends Mixins(
   }
 
   /** Called when alteration summary data has changed. */
-  protected async onAlterationSummaryChanges (): Promise<void> {
+  async onSpecialResolutionSummaryChanges (): Promise<void> {
```

Comment:
I was getting some errors in the codebase. so changed to public. I think this will may be outside the class, What would you prefer?
![Screen Shot 2022-07-25 at 12 55 04 PM](https://user-images.githubusercontent.com/55506249/180863517-7329de12-a2ee-4aab-bc04-ed5017036653.png)

Reply:
In my opinion, the template is part of this class (even though your TS is complaining).

Do we know what the future holds for property/method visibility? For example, if visibility will be enforced soon, and the template is not part of the class, then sadly this method will need to be public (but I feel strongly that it really shouldn't be exposed).

I get visibility warnings from Vetur, not TS. Should I be using your VS Code extensions, or should you be using mine?

Reply:
yeah, I think its coming from vuter Extention

Reply:
So, do you think it should be protected or public?

(I don't like the idea of this internal method being public.)

Reply:
I will update with protected